### PR TITLE
fix: set min time in calendar

### PR DESCRIPTION
**Pull request by maze-runnar**
Fixes https://github.com/fossasia/open-event-frontend/issues/4996

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

File: app/components/schedule.ts [hunk 1]
```diff
@@ -120,8 +120,14 @@ export default class Schedule extends Component<ScheduleArgs> {
    */
   adjustMinTime(view: FullCalendarView, calendar: JQuery<HTMLElement>): void {
     if (isTesting || !(view.type === 'agendaDay' || view.type === 'timelineDay')) {return}
-
-    let minTime = '0:00:00';
+    let min_time = '24:00:00';
+    this.args.sessions.map(x => {
+      const z = String(x.startsAt.format('HH:mm:ss'));
+      if (min_time > z) {
+        min_time = z;
```
Comment (iamareebjamal) [inline app/components/schedule.ts:127]: https://stackoverflow.com/questions/1669190/find-the-min-max-element-of-an-array-in-javascript#6102340

Comment (maze-runnar) [reply app/components/schedule.ts:127]: this is not working on string values

**Comment by mariobehling** [comment]
What does this exactly solve? Which minimum time? For the scheduler? We had defined 5 minutes previously. Do you plan change here?

Please always add screenshots to areas you work on. This makes everything easier.

**Comment by maze-runnar** [comment]
it sets min starting time in calendar based on session starting time.

**Review by sachinchauhan2889 — COMMENTED** [review]
it is working fine for public page. suppose my event start from 21 jan 00:00 am to 21 jan 23:00 pm. My first session is from 07:00 am. At public page, calendar starts from 07:00 am.
But this is problematic for organizers. At scheduler page also calendar starts from 07:00 am.
example: my sessions start from 00:00 am to 23:00 pm. I want to schedule my sessions. calendar at scheduler page starts from 00:00 am to 23:00pm. I schedule my first session at 07:00 am. Now after this calendar at scheduler show time from 07:00 am to 23:00pm. Now I can't schedule any sessions before 07:00 am as schduler calendar is starting from 07:00 am.

## public looks good:-
public page

## scheduler 
scheduler

**Comment by iamareebjamal** [comment]
Thanks @sachinchauhan2889

**Review by sachinchauhan2889 — APPROVED** [review]
works for me.

**Pull request merged**